README.md

```diff
@@ -46,6 +46,15 @@ If you have any questions about, feedback for or problems with flintlock:
 
 Your feedback is always welcome!
 
+## Compatibility
+
+The table below shows you which versions of Firecracker are compatible with Flintlock:
+
+| Flintlock         | Firecracker        |
+| ----------------- | ------------------ |
+| <= v0.1.0-alpha.8 | <= v0.25.2-macvtap |
```

Comment:
add a note that alpha.7 will not work with anything? or we can delete 7 i guess

Reply:
Thats a good point...i'll add a note